### DISCUSSION TITLE
I am here

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#55] [Announcements]
Sorry, I am going to Ukraine by car and could not answer or add anything. I have to come to Ukraine tomorrow (I will be able to use the computer) and today I still have the opportunity to do something on the computer. Now I have an error in the code and there is still a problem with the need to redesign the save system because when I added different pins and created a chip, in the center of the screen appeared just a square.

**Comment by firecerne** [comment]
I'll fix the save system. One day maybe I will try to do something more exciting, but first I need to do Edit Chips and rework save system.

**Reply by firecerne** [reply]
Maybe the distinction between input and output scripts was necessary. I think it would be easier to build a new system on that instead of redoing the save system.

**Comment by AlphaCodingPilot** [comment]
Have we fixed the issue of the git conflicts between the copy-paste system and the save system?

**Reply by firecerne** [reply]
No, but I'm working on it. I think it wasn't merge issues but just problems with the new signal input/output system.

**Comment by UkrainianBanderasCat** [comment]
[https://itch.io/post/6105554](url)

**Reply by UkrainianBanderasCat** [reply]
Check it.
